ETC.tsv

```diff
@@ -1698,7 +1698,7 @@ ETC_20150317_001697	Place the item you want to enhance.
 ETC_20150317_001698	{memo X}Card Enhancement
 ETC_20150317_001699	Gem Enhancement
 ETC_20150317_001700	You can no longer equip it. 
-ETC_20150317_001701	The item has been destroyed due to the death penalty.
+ETC_20150317_001701	Your equipment has been damaged due to the death penalty.
```

Comment:
If it's both, then shouldn't it say "Your equipment has been damaged and your gems have dropped on the ground due to the death penalty."